frontend/src/views/Main/components/Sidebar/components/ChatListItem/ChatListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./ChatListItem.module.css";
+
+const ChatListItem = () => {
+	return (
+		<span className={styles.listItem}>
+			<span className={styles.unRead}>
+				<div />
+			</span>
+			<img src="/images/icons/icons8-person-100.png" />
```

Comment:
Is there a way to abstract images like these icons into their own components so they can easily be reused? and maybe set a couple sizing props on those components so that they can be resized as well?

Reply:
Do you mean like the img tag itself?

Reply:
Yeah like take the image tags for these icons, wrap them in a component function and then pass some custom props to them. I can show you an example in a little bit.

Reply:
Something like this @eliotmjohnson 

![code](https://github.com/johnsonav1992/jbros-chat/assets/97169070/6691ad72-68bf-429b-bba0-1a95f2849bc6)